src/main/resources/reference.conf

```diff
@@ -869,6 +869,29 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  catalog = {
```

Comment:
Currently all the actions have to do with catalog metadata. Could we name this `catalog_metadata`?

Reply:
Hmmm since there will only be one of these resources per catalog instance, `catalog` seems clearer to me